### PR TITLE
Ansible-playbook for on-prem openshift cluster creation.

### PR DESCRIPTION
**Pull request by shashank855**
**What this PR does / why we need it**:
- Ansible-playbook for on-prem openshift cluster creation.
- Ansible-playbook `delete_cluster.yml` for deleting the cluster.

**Special notes for your reviewer**:
- Ansible-playbook `create_openshift_cluster.yml` consumes file ip.csv to ssh inside the VMs and do the pre-istallation settings.
- The VMs corresponds to the IPs in ip.csv file will have all the pre-requisites installed for openshift-cluster creation. 
- Each VMs should have snapshot after installing all the dependency.
- Ansible playbook `create_cluster.yml` does the following thing:
    - Registers VMs with redhat subscription
    - Setup the passwordless ssh connection between the VMs (which is going to be part of cluster).
    - Joins the VMs to the central DNS server.
    - Finally Running the installation playbook for cluster-creation.
    - Makes post installation settings to deploy OpenEBS.
- Ansible-playbook `delete_cluster.yml` uses `vm_name.csv` file to get the VM IDs and the corresponding snapshot ID whose values are used to revert the VMs to the previous snapshot.
- Ansible-playbook `delete_cluster.yml` does the following things:
    - Unregisters the VMs form redhat subscription.
    - Unregisters the VMs form central DNS server.
    - Finally Reverts the VMs to its initial snapshot (snapshot which is taken before after installing all the dependency on VMs).

### CODE REVIEW

File: k8s/on-prem/openshift-installer/create_openshift_cluster.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+- hosts: localhost
+
+  vars_files:
+    - vars.yml
+  
+  vars:
+    vpc_name:
```
Comment (nsathyaseelan) [inline k8s/on-prem/openshift-installer/create_openshift_cluster.yml]: @shashank855 what is the vpc_name here? is anywhere your are using it?

Comment (shashank855) [reply k8s/on-prem/openshift-installer/create_openshift_cluster.yml]: @nsathyaseelan I made the change.

File: k8s/on-prem/openshift-installer/delete_cluster.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+- hosts: localhost
+
+  vars_files:
+    - vars.yml
+  
+  vars:
+    vpc_name:
```
Comment (nsathyaseelan) [inline k8s/on-prem/openshift-installer/delete_cluster.yml]: @shashank855 remove this vpc_name vars also?

File: k8s/on-prem/openshift-installer/create_openshift_cluster.yml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        - name: Copying inventory into master
+          shell: scp -o StrictHostKeyChecking=no inventory.yml root@{{ master_ip.stdout }}:/root/openshift-ansible/inventory/
+
+        - name: Checking out to release branch-3.10
+          shell: ssh -o StrictHostKeyChecking=no root@{{ master_ip.stdout }} 'cd /root/openshift-ansible && git checkout release-3.10'
+
+        - name: Running pre-requisite.yml
+          shell: ssh -o StrictHostKeyChecking=no root@{{ master_ip.stdout }} 'ansible-playbook -i /root/openshift-ansible/inventory/inventory.yml /root/openshift-ansible/playbooks/prerequisites.yml -vv'
+          
+        - name: Running deploy-cluster.yml
```
Comment (gprasath) [inline k8s/on-prem/openshift-installer/create_openshift_cluster.yml]: Change the task names. Exclude file names.

File: k8s/on-prem/openshift-installer/create_openshift_cluster.yml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+          shell: |
+            ssh -o StrictHostKeyChecking=no root@{{item}} 'echo {{ master_key.stdout }} >> ~/.ssh/authorized_keys'
+            eval 'ssh-agent' 
+          with_items: "{{ compute_ip.stdout_lines }}"
+ 
+        - name: SSH from master to each Nodes 
+          shell: ssh -o StrictHostKeyChecking=no root@{{ master_ip.stdout }} 'ssh -o StrictHostKeyChecking=no root@{{item}} ls'
+          with_lines: cat ./ip.csv
+
+        - name: Gerating the random number
```
Comment (gprasath) [inline k8s/on-prem/openshift-installer/create_openshift_cluster.yml]: typo

File: k8s/on-prem/openshift-installer/create_openshift_cluster.yml [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+            ssh -o StrictHostKeyChecking=no root@{{item.1}} 'systemctl start systemd-hostnamed'  
+          with_together: 
+            - [ '1', '2', '3' ]
+            - "{{ compute_ip.stdout_lines }}" 
+
+        - name: Unsubscribing the nodes
+          shell: ssh -o StrictHostKeyChecking=no root@{{item}} 'subscription-manager unregister'
+          ignore_errors: true
+          with_lines: cat ./ip.csv 
+
```
Comment (gprasath) [inline k8s/on-prem/openshift-installer/create_openshift_cluster.yml]: Give comments

Comment (shashank855) [reply k8s/on-prem/openshift-installer/create_openshift_cluster.yml]: @gprasath I have made the changes.

**Pull request merged**